core/src/test/scala/unit/kafka/server/AlterUserScramCredentialsRequestNotAuthorizedTest.scala

```diff
@@ -67,15 +75,25 @@ class AlterUserScramCredentialsRequestNotAuthorizedTest extends BaseRequestTest
 
     val results = response.data.results
     assertEquals(2, results.size)
-    checkAllErrorsAlteringCredentials(results, Errors.CLUSTER_AUTHORIZATION_FAILED, "when not authorized")
+    if(!isKRaftTest())
+      checkAllErrorsAlteringCredentials(results, Errors.CLUSTER_AUTHORIZATION_FAILED, "when not authorized")
+    else{
+      checkErrorAlteringCredentials(results, Errors.RESOURCE_NOT_FOUND, 1, "when not authorized")
```

Comment:
These should also get `Errors.CLUSTER_AUTHORIZATION_FAILED` error. looks like authorizer is not set on controller nodes. check similar test AlterUserScramCredentialsRequestTest